Microsoft.Toolkit.Uwp/Microsoft.Toolkit.Uwp.csproj

```diff
@@ -48,11 +48,14 @@
       <SubType>Designer</SubType>
     </None>
     <None Include="project.json" />
+    <None Include="stylecop.json" />
+    <None Include="stylecop1.json" />
```

Comment:
I think the `stylecop.json` files don't belong to this PR. Please remove the lines from the project file.